conjur/controller/init_controller.py

```diff
@@ -137,13 +148,17 @@ def _get_server_certificate(self, conjur_url: ParseResult) -> Optional[str]:
         return fetched_certificate
 
     # pylint: disable=line-too-long,logging-fstring-interpolation,broad-except,raise-missing-from
-    def _get_account_info(self):
+    def _get_account_info_if_not_exist(self):
         """
         Method to fetch the account from the user
         """
         if self.conjurrc_data.conjur_account is None:
             try:
-                self.init_logic.fetch_account_from_server(self.conjurrc_data)
+                self.init_logic.fetch_account_from_server(self.conjurrc_data, self.ssl_verification_data)
+            except HttpSslError as ssl_err:
+                raise HttpSslError("SSL Error. Make sure Conjur "
+                                   "server's root certificate is trusted in this machine. "
+                                   "If this problem continues, visit docs fo more help") from ssl_err
```

Comment:
@shulifink

Reply:
"Make sure the Conjur certificate is trusted on this machine. If this issue persists, see <where in docs??>"

Where in the docs should we refer them to? Even if it's not an exact place, we should state where about in the docs? Alternatiavley, we  could we refer them to customer support...